### ISSUE TITLE
Variables are not autocompleted

### ISSUE THREAD

**Issue by tdi** [erd0s/terraform-autocomplete#3]
Would be great if variables and items from other files were suggested in completion.

**Comment by erd0s** [comment]
Definitely! This is on the top of the list with a few other things next time I get a chance to work on it, in the mean time PRs definitely accepted
